easybuild/easyconfigs/c/Celera_Assembler/Celera_Assembler-8.1-goolf-1.4.10.eb

```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# Swiss Institute of Bioinformatics (SIB)
+
+easyblock = 'MakeCp'
+
+name = 'Celera_Assembler'
+version = '8.1'
+
+homepage = 'http://wgs-assembler.sourceforge.net/'
+description = """ Celera Assembler is a de novo whole-genome shotgun (WGS) DNA 
+ sequence assembler. It reconstructs long sequences of genomic DNA from fragmentary 
+ data produced by whole-genome shotgun sequencing. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = [('http://sourceforge.net/projects/wgs-assembler/files/wgs-assembler/wgs-%(version)s/', 'download')]
+sources = ['wgs-%(version)s.tar.bz2']
+
+parallel = 1
+
+prebuildopts = 'cd kmer && make install && cd .. && '
+prebuildopts += 'cd samtools && make && cd .. && '
+prebuildopts += 'cd src && '
+
+files_to_copy = ["*"]
```

Comment:
so, use `ConfigureMake` with `skipsteps = ['install']` and `buildininstalldir = True`?